assets/src/blocks/HappyPoint/HappyPointBlock.js

```diff
@@ -1,14 +1,14 @@
-import {HappypointEditor} from './HappypointEditor';
-import {HappypointBlock as HappypointBlockV1} from './deprecated/HappypointBlocklV1.js';
+import {HappyPointEditor} from './HappyPointEditor.js';
+import {HappyPointBlock as HappyPointBlocklV1} from './deprecated/HappyPointBlocklV1.js';
```

Comment:
There is a typo in this file name: `HappyPointBlocklV1` => `HappyPointBlockV1`

Reply:
Fixed!